### PR TITLE
Tweaks to "About the Automation Service"

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) makes minor updates to this article:
https://help.sumologic.com/docs/cse/automation-service/about-automation-service/#support-and-compliance

Tweaks made:
- Adds a "Data retention" section.
- Adds a statement that Terraform doesn't support creating or modifying integrations, playbooks, or actions.

Issue number:  None

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/cse/automation-service/about-automation-service.md [hunk 1]
```diff
@@ -172,8 +172,12 @@ The [CSE API](/docs/cse/administration/cse-apis/) supports automations. Endpoint
 * `GET /automations/{id}`. Get a specific automation
 * `PUT /automations/{id}`. Update a specific automation
 
-The Sumo Logic Terraform provider also supports automations. For more information, see the [Sumo Logic Terraform documentation](https://registry.terraform.io/providers/SumoLogic/sumologic/latest/docs).
+The Sumo Logic Terraform provider also supports automations, but does not support the ability to create or modify integrations, playbooks, nor actions. For more information about Terraform, see the [Sumo Logic Terraform documentation](https://registry.terraform.io/providers/SumoLogic/sumologic/latest/docs).
```
Comment (JainM6) [inline docs/cse/automation-service/about-automation-service.md:175]:
```suggestion
The Sumo Logic Terraform provider also supports automation, but does not support the ability to create or modify integrations, playbooks, or actions. For more information about Terraform, see the [Sumo Logic Terraform](https://registry.terraform.io/providers/SumoLogic/sumologic/latest/docs) documentation.
```

Comment (JainM6) [reply docs/cse/automation-service/about-automation-service.md:175]: Just a quick fix.

**Review by JainM6 — APPROVED** [review]
just minor changes.

**Review by JainM6 — CHANGES_REQUESTED** [review]
Minor changes

**Comment by jpipkin1** [comment]
Thanks for the review, @JainM6!

**Comment by jpipkin1** [comment]
@JainM6, could you check again and approve if it's ready?

**Review by JainM6 — APPROVED** [review]

**Comment by JainM6** [comment]
> @JainM6, could you check again and approve if it's ready?

 Thanks @jpipkin1. Sorry, I completely missed re-reviewing this one.

**Comment by jpipkin1** [comment]
Thank you, @JainM6!

**Pull request merged**